apps/wallets/smart-wallet/react/README.md

```diff
@@ -1,16 +1,54 @@
-# SCW DEMO (React Starter)
+# Smart Wallets Demo (React Starter Kit)
 
-Demo app to show Abstract Accounts functionality.
-This branch only uses W3A
+This application demonstrates the use of Abstract Accounts with a built-in passkey authenticator. Simply sign in with Google, authorize your passkey, and create your smart wallet in seconds.
 
-## Environment Variables
+## Prerequisites
 
-Copy the `.env.sample` file to `.env` and fill in the following:
+Before you begin, ensure you have the following installed:
 
-## Start the app
+-   Node.js (v20.12 or later)
+-   npm (comes with Node.js)
+-   pnpm (`npm install -g pnpm`)
 
-To start the app, run:
+After installing the prerequisites, clone the repository and install the dependencies:
 
 ```bash
-pnpm run start
+git clone <repository-url>
+pnpm i
 ```
+
+## Configuration
+
+Before running the application, you need to set up your environment variables:
+
+1. Copy the `.env.example` file in the root directory and rename the copy to `.env`.
```

Comment:
I would clarify that "root directory" is the directory in which this README.md is contained